blocks/table-of-contents-override/table-of-contents-override.js

```diff
@@ -0,0 +1,42 @@
+import { getLibs } from '../../scripts/utils.js';
```

Comment:
Is this file ported from blog?  If not I don't think we want to use jQuery style `$variables`

Reply:
good catch! I removed the jQuery naming pattern in https://github.com/adobecom/blog/pull/15/commits/c7366cfd7bdb67ff376dd7a1e5327438b85b6616